src/main/java/com/google/graphgeckos/dashboard/reader/BuildBotSetUpData.java

```diff
@@ -0,0 +1,32 @@
+package com.google.graphgeckos.dashboard.reader;
+
+public class BuildBotSetUpData {
+
+  // Name of the BuildBot
+  public final String name;
+
+  // Build id from where to start fetching.
+  public final long initialId;
+
+  // Request frequency in seconds.
+  public final long delay;
+
+  public BuildBotSetUpData(String name, long initialId, long delay) {
+    this.name = name;
+    this.initialId = initialId;
+    this.delay = delay;
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
We'd normally use `AutoValue` for something like this: https://github.com/google/auto